src/field_theory/polynomial_galois_group.lean

```diff
@@ -418,8 +418,9 @@ begin
     (show (gal_action_hom p ℂ).range = ⊤, from _)).mpr (subgroup.mem_top x)⟩,
   apply equiv.perm.subgroup_eq_top_of_swap_mem,
   { rwa h1 },
-  { rw [h1, ←h2],
-    exact prime_degree_dvd_card p_irr p_deg },
+  { rw h1,
+    convert prime_degree_dvd_card p_irr p_deg using 1,
+    convert h2.symm },
```

Comment:
What were the two incompatible instances here?

Reply:
it looked to me like a fintype thing. I didn't debug it, `exact h2.symm` didn't work and `convert` did, it's just one of those things.

Reply:
I just had a look on gitpod; it's a conflict between "the range of a morphism from a finite type is finite" and "a subgroup of a finite type is finite". I think it's worth adding a comment to each of your new instances indicating the instance that they can form a diamond with.